regenerate.sh

```diff
@@ -15,16 +15,16 @@
 
 set -eu -o pipefail
 
-WORKDIR=$(mktemp -d)
+export WORKDIR=$(mktemp -d)
 
 function finish {
   rm -rf "$WORKDIR"
 }
 trap finish EXIT
 
-export GOBIN=${WORKDIR}/bin
-export PATH=${GOBIN}:${PATH}
-mkdir -p ${GOBIN}
+export GOBIN="${WORKDIR}"/bin
```

Comment:
```suggestion
export GOBIN="${WORKDIR}/bin"
```
See: go/bash-style#s5.7-quoting